lib/LTP/utils.pm

```diff
@@ -49,7 +49,7 @@ sub loadtest_runltp {
     my %env = %{$tinfo->test_result_export->{environment}};
     $env{retval} = 'none';
 
-    if (is_test_disabled(\%env, $tinfo->runfile, $name)) {
+    if (get_var('LTP_KNOWN_ISSUES') && is_test_disabled(\%env, $tinfo->runfile, $name)) {
```

Comment:
From my POV I would change `LTP::WhiteList` that the variable isn't required anymore.

Reply:
OK, moved check and retested. Feel free to have a look.